app/models/foreman_openscap/policy_revision.rb

```diff
@@ -1,6 +0,0 @@
-module ForemanOpenscap
-  class PolicyRevision < ApplicationRecord
-    belongs_to :policy
-    belongs_to :scap_content
```

Comment:
is this related? is there a leftover table that we should drop?

Reply:
You are right, migration will be [needed](https://github.com/theforeman/foreman_openscap/blob/master/db/migrate/20141104171545_create_scaptimony_policy_revisions.rb). I will raise a separate issue.